### PR TITLE
Docs Feedback

### PR DESCRIPTION
**Pull request by calebeby**

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -279,8 +295,12 @@ export const getStarredEvents = (userId: number) =>
 // Anyone can modify themselves
 // Only admins can modify other users
 export const modifyUser = (userId: number, user: EditableUser) =>
-  putRequest<null>(`/users/${userId}`, user)
+  patchRequest<null>(`/users/${userId}`, user)
```
Comment (BrendanBurkhart) [inline src/api.ts]: Is this patching with the full user? Because that basically defeats the purpose of the patch, and makes it a put.

Comment (calebeby) [reply src/api.ts]: No, it doesn't have to be a full user. I forgot to switch this, cause there was a bug in the docs generator. I will add that.

File: docs/docs.md [hunk 2]
```diff
@@ -491,14 +513,17 @@ Anyone can view any user
 ```ts
 type Data = {
   username: string
-  name: string
+  name: {
```
Comment (fharding1) [inline docs/docs.md]: I don't really like this... can we just do firstName and lastName?

File: docs/docs.md [hunk 3]
```diff
@@ -508,9 +533,15 @@ Only admins can modify other users
 ```ts
 type Data = {
```
Comment (fharding1) [inline docs/docs.md]: Everything here needs to be optional...

File: src/api.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 type PeregrineResponse<T> =
```
Comment (fharding1) [inline src/api.ts]: {"data": {}, "error": {"error": "bla", "code": 4}}

File: docs/docs.md (hunk 2 above)
Comment (calebeby) [reply docs/docs.md]: :heavy_check_mark:

File: src/api.ts (hunk 4 above)
Comment (calebeby) [reply src/api.ts]: :heavy_check_mark:

File: docs/docs.md (hunk 3 above)
Comment (calebeby) [reply docs/docs.md]: :heavy_check_mark:

**Pull request merged**